### ISSUE TITLE
Compatibility issue with deb-repo-poller

### ISSUE THREAD

**Issue by hulu1522** [gocd-contrib/gem-repo-poller#1]
It seems that this plugin and your [deb-repo-poller](https://github.com/srinivasupadhya/deb-repo-poller) plugin have the same ID and Go-Server fails to load them because of that.

```
java.lang.RuntimeException: Found another plugin with ID: deb
        at com.thoughtworks.go.plugin.infra.plugininfo.DefaultPluginRegistry.loadPlugin(DefaultPluginRegistry.java:41)
        at com.thoughtworks.go.plugin.infra.listeners.DefaultPluginJarChangeListener.addPlugin(DefaultPluginJarChangeListener.java:108)
        at com.thoughtworks.go.plugin.infra.listeners.DefaultPluginJarChangeListener.pluginJarAdded(DefaultPluginJarChangeListener.java:72)
        at com.thoughtworks.go.plugin.infra.monitor.DefaultPluginJarLocationMonitor$PluginLocationMonitorThread$DoOnAllListeners$1.execute(DefaultPluginJarLocationMonitor.java:
266)
        at com.thoughtworks.go.plugin.infra.monitor.DefaultPluginJarLocationMonitor$PluginLocationMonitorThread$DoOnAllListeners.doOnAll(DefaultPluginJarLocationMonitor.java:29
7)
        at com.thoughtworks.go.plugin.infra.monitor.DefaultPluginJarLocationMonitor$PluginLocationMonitorThread$DoOnAllListeners.pluginJarAdded(DefaultPluginJarLocationMonitor.
java:264)
        at com.thoughtworks.go.plugin.infra.monitor.DefaultPluginJarLocationMonitor$PluginLocationMonitorThread.notifyListenersOfAddedPlugins(DefaultPluginJarLocationMonitor.java:195)
        at com.thoughtworks.go.plugin.infra.monitor.DefaultPluginJarLocationMonitor$PluginLocationMonitorThread.loadAndNotifyPluginsFrom(DefaultPluginJarLocationMonitor.java:186)
        at com.thoughtworks.go.plugin.infra.monitor.DefaultPluginJarLocationMonitor$PluginLocationMonitorThread.run(DefaultPluginJarLocationMonitor.java:172)
```

**Comment by hulu1522** [comment]
Update:  Looks like your source code has the correct ID but for some reason the [compiled .jar](https://github.com/srinivasupadhya/gem-repo-poller/releases) has the `deb` ID and was fixed in commit f2e36d8d8d36d28156646c6426f3ec3840fe145e.  You may need to compile and release another version with the updated ID.

**Comment by srinivasupadhya** [comment]
fixed that. sorry for the trouble.
